frontend/src/components/Vote.js

```diff
@@ -27,28 +27,30 @@ VoteDialog.propTypes = {
   isAuthzMode: PropTypes.bool.isRequired,
 };
 
-
 export default function VoteDialog(props) {
   const [option, setOption] = React.useState("");
   const dispatch = useDispatch();
-  const [granter, setGranter] = React.useState(props.granters.length > 0 ? props.granters[0] : "");
+  const [granter, setGranter] = React.useState(
+    props.granters.length > 0 ? props.granters[0] : ""
```

Comment:
I don't see **granters** as a required prop. And if granters is undefined, the whole page will break. It good to practice destructuring the object in the beginning and have some default values. 
`const {granters = []} = props;`